week03/css/contact.css

```
@@ -0,0 +1,205 @@
+@font-face {
+    font-family: '국립박물관문화재단클래식B';
+    src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_twelve@1.0/국립박물관문화재단클래식B.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: '국립박물관문화재단클래식M';
+    src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_twelve@1.0/국립박물관문화재단클래식M.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+#header {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    padding: 10px 15px;
+    background-color: white;
+    color: #6a75ca;
+    font-size: 18px;
+    border-bottom: 3px solid #B4C7E7;
+}
+
+body {
+    background-color:#E6EEFB;
+}
+
+.title {
+    font-family: '국립박물관문화재단클래식B';
+    font-size: 40px;
+    color: black;
+    justify-content: center;
+    text-align: center;
+    display: flex;
+    margin: 250px 0 0 0;
+}
+
+.mail {
+    font-family: '국립박물관문화재단클래식M';
+    font-size: 20px;
+    color: black;
+    text-align: center;
+    display: inline-block;
+    margin: 150px 0 0 350px;
+}
+
+.t1 {
+    font-family: '국립박물관문화재단클래식B';
+    font-size: 35px;
+    color: #0D51FE;
+    margin: 0 0 50px 0;
+}
+
+.mail img {
+    margin: 30px 0 0 0;
+}
+
+.github {
+    font-family: '국립박물관문화재단클래식M';
+    font-size: 20px;
+    color: black;
+    text-align: center;
+    display: inline-block;
+    margin: 150px 0 0 210px;
+}
+
+.t2 {
+    font-family: '국립박물관문화재단클래식B';
+    font-size: 35px;
+    color: #0D51FE;
+    margin: 0 0 50px 0;
+}
+
+.github img {
+    margin: 30px 0 0 0;
+}
+
+.tistory {
+    font-family: '국립박물관문화재단클래식M';
+    font-size: 20px;
+    color: black;
+    text-align: center;
+    display: inline-block;
+    margin: 150px 0 0 210px;
+}
+
+.t3 {
+    font-family: '국립박물관문화재단클래식B';
+    font-size: 35px;
+    color: #0D51FE;
+    margin: 0 0 50px 0;
+}
+
+.tistory img {
+    margin: 30px 0 0 0;
+}

```

1. Tistory 요소 위치 오류를 해결했습니다.
2. 코드 재활용성을 높이는 것이 좋습니다.

```suggestion
body main ul {
    padding: 0;
    display: flex;
    flex-flow: row;
    justify-content: space-evenly;
    list-style-type: none;
}
body main ul li {
    display: flex;
    flex-flow: column;
}
body main ul li span {
    text-align: center;
    font-family: '국립박물관문화재단클래식B';
    font-size: 35px;
    color: #0D51FE;
}
body main ul li a {}
body main ul li a img {
    width: 150px;
    padding-top: 20px;
}
```